chainerx_cc/chainerx/routines/statistics.cc

```diff
@@ -7,14 +7,21 @@
 #include "chainerx/backward.h"
 #include "chainerx/backward_builder.h"
 #include "chainerx/backward_context.h"
+#include "chainerx/dtype.h"
 #include "chainerx/macro.h"
 #include "chainerx/routines/creation.h"
+#include "chainerx/routines/type_util.h"
 
 namespace chainerx {
 
+Dtype PromoteInt2Float(Dtype dtype){
+    return GetKind(dtype) == DtypeKind::kFloat? dtype :internal::GetDefaultDtype(DtypeKind::kFloat);
+}
```

Comment:
Seems like this function should be clang-formatted (to a single line).

Reply:
Updated.